cluster-autoscaler/go.mod

```diff
@@ -55,15 +55,18 @@ require (
 	k8s.io/kubernetes v1.29.6
 	k8s.io/legacy-cloud-providers v0.0.0
 	k8s.io/utils v0.0.0-20231127182322-b307cd553661
-	sigs.k8s.io/cloud-provider-azure v1.29.0
+	sigs.k8s.io/cloud-provider-azure v1.29.4
```

Comment:
The latest is 1.29.9, do we plan to bump it separately from this PR?

Reply:
(discussed offline) separately